### PR TITLE
401-make-SSL-support-configurable

### PR DESCRIPTION
**Pull request by divino**
#401

### CODE REVIEW

File: app/templates/bower.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
     "lodash": "~3.10.1",
     "ml-search-ng": "~0.2.0",
     "tinymce-dist": "4.3.12",
-    "vkbeautify-wrapper": "*",
-    "ml-highcharts-ng": "~0.0.11"
```
Comment (grtjn) [inline app/templates/bower.json]: Please undo these changes..

File: app/templates/gulpfile.js [hunk 2]
```diff
@@ -949,10 +949,10 @@ function getNodeOptions(env) {
       '`gulp init-local` creates local.json which can be modified for other environments as well');
   }
   var port = args['app-port'] || process.env.PORT || envJson['node-port'] || config.defaultPort;
-  return {
-    script: config.nodeServer,
-    delayTime: 1,
-    env: {
+
+  console.log("config.nodeServer : " + config.nodeServer);
```
Comment (grtjn) [inline app/templates/gulpfile.js]: Could you either make this a more elegant message (if this info is not yet printed somewhere yet), or just get rid of it?

**Comment by grtjn** [comment]
Looks really promising!

I made two inline comments, and would like to ask you to rebase your work on cards-theme branch, and  open a PR against that branch. I can then merge, and let CI run some (automated) tests with it..

**Comment by grtjn** [comment]
@daveegrant happens to have a need for this, holding off a little to see if he his feedback..

**Comment by grtjn** [comment]
We have a merge conflict because of changes in PR #449.

**Comment by grtjn** [comment]
@divino Would you have time to look into rebasing, and adjusting proxy.js to work on top of the changes Joe did on proxy.js?

**Comment by divino** [comment]
@grtjn sure.

**Comment by joemfb** [comment]
The `http-proxy` module should help make it a little easier: https://github.com/nodejitsu/node-http-proxy#using-https

**Comment by divino** [comment]
@joemfb Thanks

**Comment by grtjn** [comment]
@divino, were you still running tests, or do you think this is ready to merge? It looks like you managed to cleanly merge proxy.js.. :)

**Comment by divino** [comment]
@geert - I need to test it with production type certificates IMO although It worked well with self signed certificates when I tested it.

**Comment by grtjn** [comment]
Runs fine out of the box, merging into master. If https/ssl needs further tweaks, please open a new PR..
